src/components/ConfirmMonthModal.js

```diff
@@ -0,0 +1,63 @@
+const ConfirmMonthModal = ({
+  setIsModalOpen,
+  setSelectedMonth,
+  monthToShow,
+}) => {
```

Comment:
personal opinion: breaking up the parameters into different lines makes the code very hard to read.

Reply:
No disrespect meant but I actually love it 😄 ... especially when the props are more than 3. 
Prettier does its formatting magic here ... @MatthewBerglund any opinion on this? Any preference? I could investigate how to change this behaviour in Prettier if the majority is against. No problem from my side.

Reply:
I guess my personal preference would be to have function params on a single line, but it's not a must for me. Also, we'd probably need to reformat every file at this point, since I'm sure many of our components are formatted this way.

Reply:
After investigating a bit, it looks like to me that we can change this by overriding the print-width in the prettier config. I'd ask you to play around with it a bit and decide on a print-width you like to implement. Let me know!

Reply:
i think we are all grown-ups and can break our own lines :) I would disable the rule entirely.

personally, I do like linters, because they make sure you are on the same page in terms of style. however, I prefer to configure my linters very forgiving. at the end of the day, the developer should be able to decide what looks good and not a machine.